combination.jmd

```diff
@@ -53,170 +53,123 @@ pal = DrugResponseModel.getODEparams(pal_before, concs[:, 5]);
 ```
 
 # Combination
-
 ```julia; echo=false
+
 # Bliss on pure cell numbers
-BlissOnData = DrugResponseModel.blissCellNum(g1avg, g2avg; T = 96, n = 8) # includes are combinations 8 x 8 x 10
-
-""" This function calculates cell number for parameter sets that are the result of Bliss on prog. rates. """
-function BlissModelComb(bliss_comb, g0)
-      bliss_comb_cellnum = zeros(8,8)
-      for i=1:8
-      for j=1:8
-            g1, g2, _ = predict(bliss_comb[:, i, j], g0, 96)
-            bliss_comb_cellnum[i,j] = g1 + g2
-      end
-      end
-      return bliss_comb_cellnum
-end
-
-""" This function plots the heatmap of combined cell numbers given any two drugs. """
-function Heatmap(concs, data, i1, i2, d1name, d2name, title)
-    concs[1, :] .= 0.05
-    heatmap(
-        string.(round.(log.(concs[:, i2]), digits = 1)),
-        string.(round.(log.(concs[:, i1]), digits = 1)),
-        data,
-        xlabel = string(d2name, " log[nM]"),
-        ylabel = string(d1name, " log [nM]"),
-        title = title,
-        clim = (0.0, 40.0),
-    )
-end
+BlissOnData = DrugResponseModel.blissCellNum(g1avg, g2avg; T = 189, n = 8); # includes are combinations 8 x 8 x 10
+
 ```
+
 ## Lapatinib and doxorubicin
 
 ```julia; echo=false
 bliss_comb = DrugResponseModel.fullCombinationParam(lapat, dox, effs, 8)
-bliss_comb_cellnum = BlissModelComb(bliss_comb, g0)
-
-loeweOnData = DrugResponseModel.loweCellNum(concs, 1, 2, g1avg, g2avg);
+bliss_comb_cellnum = BlissModelComb(bliss_comb, g1avg, g2avg, 1, 2)
```

Comment:
If `BlissModelComb` is a model prediction, why does it need the data?

Reply:
If it's for scaling then pass the scale, not the entirety of the data. It's important to reduce the amount of data passed to a function to exactly what it needs, to ensure it's not dependent upon things you don't expect.

Reply:
Edited. Now it is only getting `bliss_comb, g0` as input.